apps/intellij/src/main/kotlin/dev/nx/console/angular/NxAngularProject.kt

```diff
@@ -0,0 +1,80 @@
+package dev.nx.console.angular
+
+import com.intellij.lang.Language
+import com.intellij.lang.css.CSSLanguage
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import org.angular2.cli.config.AngularJsonProject
+import org.angular2.cli.config.AngularLintConfiguration
+import org.angular2.cli.config.AngularProject
+
+class NxAngularProject(
```

Comment:
This class not only provides context determination, but also some metadata for WebStorm to handle Angular project specifics, like global stylesheets, better.